interface/modules/custom_modules/oe-module-comlink-telehealth/public/index-portal.php

```diff
@@ -24,7 +24,11 @@
 // need to retain the webroot if we have one
 $landingpage = $basePath . "portal/index.php?site=" . urlencode($_GET['site_id'] ?? '') . "&redirect=" . urlencode($redirect);
 $skipLandingPageError = true;
-
+// test session
+if (!empty($_REQUEST['me'] ?? null) && empty($_SESSION ?? null)) {
+    session_id($_REQUEST['me']);
+    session_start();
+}
```

Comment:
is this needed? Would be ideal to avoid passing session ids in request parameters.
(the code in verify_session should capture a portal session, so odd that this is needed)

Reply:
agreed @bradymiller I don't know if this is a window issue or a browser issue. Currently testing with chrome engine.
The verify session doesn't work. I had this same issue with the CCDA rest call from portal. This was the solution and has been in code for at least 4 years. Ran into same issue with OAuth but there I needed to store the session in database because of a different origin for request.
It is not unusual to use session id in session management. Still, maybe will encrypt id.
 or store to database to be more comfortable.
Since this is a same origin request the session should be intact or maybe something in my PHP settings.
Will get Stephen to verify he doesn't see the session on his environment.

Reply:
I'm not seeing this problem anymore with me fixes from yesterday on linux on Chrome.  I'll test this on firefox.  Perhaps its something on Windows and I can try booting up a VM to test this out.

Reply:
Let me fire up a docker and test first and maybe will save some time.